app/views/content/is-teaching-right-for-me/teach-disabled-pupils-and-pupils-with-special-educational-needs.md

```diff
@@ -9,7 +9,7 @@ image: "static/content/hero-images/0023.jpg"
 navigation: 5.40
 navigation_title: How can I teach children with special educational needs?
 navigation_description: Find out more about teaching pupils with special educational needs and disabilities.
-article_classes: ['longform']
+article_classes: ['longform', 'blog']
```

Comment:
@garethwilliamsdesign I'm not sure this is how we normally add quotes to non-blog pages? Blog quotes are formatted a bit differently I think?

See `/teacher-training-advisers` and `/is-teaching-right-for-me/teaching-internship-providers` (which I know are html not md!)